routemodel/data_retrieval/get_elevations_point_compression.py

```diff
@@ -0,0 +1,165 @@
+import requests
+import pandas as pd
+import sys
+import os.path
+from csv import reader
+from config import API_KEY
+
+sys.path.append(os.path.dirname(sys.path[0]))
+
+BASE_URL = 'https://dev.virtualearth.net/REST/v1/'
+ELEVATIONS_FILE = os.path.join(sys.path[0], '../../routemodel/routes/heartland_elevations.csv')
+COORDINATES_FILE = os.path.join(sys.path[0], '../../routemodel/routes/heartland_coordinates.csv')
+
+# This just generates a dict object for the coordinates to resolve some type errors
+COORDINATES_LISTX = []
+COORDINATES_LISTY = []
+COORDINATES_LIST = []

```

Comment:
new line here please

Reply:
I moved this section under if __init__== "__main__":